stix_shifter_modules/symantec_endpoint_security/README.md

```diff
@@ -0,0 +1,700 @@
+                    "x_is_on_premises": false
+                },
+                "4": {
+                    "type": "mac-addr",
+                    "value": "11:1X:XX:X1:11:X1"
+                },
+                "5": {
+                    "type": "x-symantec-policy",
+                    "name": "Default Antimalware Policy",
+                    "version": "1"
+                },
+                "7": {
+                    "type": "ipv6-addr",
+                    "value": "xx11::1x1x:11xx:11xx:x1x1",
+                    "resolves_to_refs": [
+                        "4"
+                    ]
+                },
+                "8": {
+                    "type": "ipv4-addr",
+                    "value": "2.2.2.2",
+                    "resolves_to_refs": [
+                        "10"
+                    ]
+                },
+                "9": {
+                    "type": "ipv6-addr",
+                    "value": "xx22::22x2:x0xx:22x2:22xx",
+                    "resolves_to_refs": [
+                        "10"
+                    ]
+                },
+                "10": {
+                    "type": "mac-addr",
+                    "value": "22:X2:22:X2:X2:22"
+                },
+                "11": {
+                    "type": "software",
+                    "name": "Windows Server 2019 Datacenter Edition",
+                    "x_os_type": "Windows"
+                },
+                "12": {
+                    "type": "ipv4-addr",
+                    "value": "3.3.3.3"
+                },
+                "13": {
+                    "type": "user-account",
+                    "user_id": "SYSTEM"
+                }
+            },
+            "number_observed": 1,
+            "last_observed": "2024-05-06T08:33:25Z",
+            "first_observed": "2024-05-06T08:33:25Z"
+        }
+    ],
+    "spec_version": "2.0"
+}
+```
+### Types of Attributes
+
+| Type                      | Description                                | Example                                                                                                            |
+|---------------------------|--------------------------------------------|--------------------------------------------------------------------------------------------------------------------|
+| Dictionary fields         | A key value pair attributes.               | {"user_name": "SYSTEM"}, {"feature_name": "MALWARE_PROTECTION"}                                                              |
+| List of dictionary fields | A list containing one or more dictionaries | "device_networks": [{"ipv4": "1.1.1.1", "ipv6": "xx22::2x2x:22xx:22xx:x2x2", "mac": "11:1X:XX:X1:11:X1" }, { "ipv4": "2.2.2.2", "ipv6": "xx11::1x1x:11xx:11xx:x1x1", "mac": "22:X2:22:X2:X2:22"}] |
+
+### Observations
+- The access token expires 60 minutes after being created.
+
+### Limitations
+- The maximum number of API calls for events is limited to 500 per hour.
```

Comment:
Do we have an error message for this?

Reply:
Yes, It is already handled by framework.

Reply:
I'd like to see a specific check made in the code for this one. 500 calls an hour is relatively low and this could be a regular issue. I'd rather be certain it's caught.

Reply:
It is handled by framework as follows: 
Error: 429 (Too many requests) is returned by data sourse server. This error is handled by framwork in function "async def call_api()" file "stix_shifter_utils/stix_transmission/utils/RestApiClientAsync.py"

Connector sends the following Error:
RuntimeError: Fetching results failed; see log for details{'success': False, 'connector': 'symantec_endpoint_security', 'error': 'symantec_endpoint_security connector error => Max retries exceeded. too_many_requests with max retry (1)', 'code': 'too_many_requests'}

Reply:
The problem with that error message is that a user won't be able to easily identify why they are erroring out. Since we know that the API will give that message at 500 calls, we should be explicit in the output that "Too many request were made in the last hour. This API is limited to 500 request per an hour."

Reply:
Added custom message for 429 error code as suggested.